PhpAmqpLib/Channel/AbstractChannel.php

```diff
@@ -170,7 +170,7 @@ public function getMethodQueue()
     /**
      * @param string $method_sig
      * @param string $args
-     * @param $content
+     * @param mixed $content
```

Comment:
I believe `$content` is always of type `AMQPMessage`.  In fact, we could probably rename the variable `$amqpMessage`